packages/block-library/src/page-list/edit.js

```diff
@@ -142,6 +160,13 @@ export default function PageListEdit( {
 		}, [] );
 	};
 
+	useEffect( () => {
+		__unstableMarkNextChangeAsNotPersistent();
+		if ( blockList ) {
+			replaceInnerBlocks( clientId, blockList );
+		}
+	}, [ clientId, parentPageID, hasResolvedPages ] );
```

Comment:
The dependencies here don't seem to match the ones used in the hook. Is there a reason why?

Reply:
`parentPageID` is a leftover, `hasResolvedPages` is not used but the hook should run I think when it changes.